Content.Client/Guidebook/DocumentParsingManager.cs

```diff
@@ -22,24 +24,29 @@ public sealed partial class DocumentParsingManager
     [Dependency] private readonly ISandboxHelper _sandboxHelper = default!;
 
     private readonly Dictionary<string, Parser<char, Control>> _tagControlParsers = new();
-    private Parser<char, Control> _tagParser = default!;
     private Parser<char, Control> _controlParser = default!;
+
+    private ISawmill _sawmill = default!;
+    private Parser<char, Control> _tagParser = default!;
     public Parser<char, IEnumerable<Control>> ControlParser = default!;
 
     public void Initialize()
     {
         _tagParser = TryOpeningTag
-            .Assert(_tagControlParsers.ContainsKey, tag => $"unknown tag: {tag}")
+            // .Assert(_tagControlParsers.ContainsKey, tag => $"unknown tag: {tag}")
             .Bind(tag => _tagControlParsers[tag]);
```

Comment:
Any particular reason this was commented out?

Reply:
I commented this out originally as I was trying to see if I could easily make it continue instead of stopping parsing at the first invalid tag error.

I had trouble implementing it so I didn't continue with the rest of the implementation.

This should be uncommented as while both (commented/uncommented) halt the parsing, the uncommented version gives a better error message.